components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -401,4 +351,18 @@ function CustomLeague:_displayMaps(maps)
 	)}
 end
 
+function CustomLeague:createLiquipediaTierDisplay(args)
+	local tierDisplay = Tier.display(
+		args.liquipediatier,
+		args.liquipediatiertype,
+		{link = true, game = args.game}
```

Comment:
```suggestion
		{link = true, game = GameLookup.getName{args.game}}
```

Reply:
could we use `Module:Game` here?

Reply:
Likely, but that could be applied to various other places and therefore is OOS